src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,86 @@
+        boolean logoutButtonIsDisplayed = logoutButton.isDisplayed();
+        Assertions.assertThat(logoutButtonIsDisplayed)
+                .isTrue();
+        //4. Assert Username is loggined
+        WebElement loginName = driver.findElement(By.id(ID_LOGIN_NAME));
+        Assertions.assertThat(loginName.getText())
+                .isEqualTo(LOGIN_ROMAN_IOVLEV);
+        //5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> itemsInHeader = driver.findElements(By.xpath(XPATH_ITEMS_IN_HEADER));
+        Assertions.assertThat(itemsInHeader.get(0).getText())
+                .isEqualTo(LIST_ITEMS_HEADER);
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> listOfImages = driver.findElements(By.className("benefit-icon"));
+        Assertions.assertThat(listOfImages.size())
+                .isEqualTo(4);
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> listOfTextUnderImages = driver.findElements(By.className("benefit-txt"));
+        Assertions.assertThat(listOfTextUnderImages.get(0).getText())
+                .isEqualTo("To include good practices\nand ideas from successful\nEPAM project");
+        Assertions.assertThat(listOfTextUnderImages.get(1).getText())
+                .isEqualTo("To be flexible and\ncustomizable");
+        Assertions.assertThat(listOfTextUnderImages.get(2).getText())
+                .isEqualTo("To be multiplatform");
+        Assertions.assertThat(listOfTextUnderImages.get(3).getText())
+                .isEqualTo("Already have good base\n(about 20 internal and\n"
+                        + "some external projects),\nwish to get more…");
+        //8.  Assert that there is the iframe with “Frame Button” exist
+        WebElement frame = driver.findElement(By.id(ID_FRAME));
+        boolean frameIsDisplayed = frame.isDisplayed();
+        Assertions.assertThat(frameIsDisplayed)
+                .isTrue();
+        //9.  Switch to the iframe and check that there is “Frame Button” in the iframe
+        driver.switchTo().frame(ID_FRAME);
+        WebElement frameButton = driver.findElement(By.id(ID_FRAME_BUTTON));
+        boolean frameButtonIsDisplayed = frameButton.isDisplayed();
+        Assertions.assertThat(frameButtonIsDisplayed)
+                .isTrue();
+        //10. Switch to original window back
+        driver.switchTo().parentFrame();
+        //11. Assert that there are 5 items in the Left Section are displayed and they have proper text
+        WebElement leftMenu = driver.findElement(By.xpath(XPATH_LEFT_MENU));
+        Assertions.assertThat(leftMenu.getText())
+                .contains("Home")
+                .contains("Contact form")
+                .contains("Service")
+                .contains("Metals & Colors")
+                .contains("Elements packs");
+    }
```

Comment:
It's a specific check like LIST_ITEMS_HEADER. Get core element and then check than it contains text is not the best way. Use 1 locator to find all 5 elements.